request.go

```diff
@@ -981,3 +1001,84 @@ func (req *ExecuteRequest) Args(args interface{}) *ExecuteRequest {
 func (req *ExecuteRequest) Body(res SchemaResolver, enc *msgpack.Encoder) error {
 	return fillExecute(enc, req.expr, req.args)
 }
+
+type PrepareRequest struct {
+	baseRequest
+	expr string
+}
+
+// NewPrepareRequest returns a new empty PrepareRequest.
+func NewPrepareRequest(expr string) *PrepareRequest {
+	req := new(PrepareRequest)
+	req.requestCode = PrepareRequestCode
+	req.expr = expr
+	return req
+}
+
+// Body fills an encoder with the execute request body.
+func (req *PrepareRequest) Body(res SchemaResolver, enc *msgpack.Encoder) error {
+	return fillPrepare(enc, req.expr)
+}
+
+type UnprepareRequest struct {
+	ConnectedObject
+	baseRequest
+	stmt *Prepared
+}
+
+// NewUnprepareRequest returns a new empty UnprepareRequest.
+func NewUnprepareRequest() *UnprepareRequest {
+	req := new(UnprepareRequest)
+	req.requestCode = PrepareRequestCode
+	req.stmt = &Prepared{}
+	return req
+}
+
+func (req *UnprepareRequest) WithPrepared(stmt *Prepared) *UnprepareRequest {
+	req.stmt = stmt
+	return req
+}
+
+func (req *UnprepareRequest) Conn() *Connection {
+	return req.stmt.Conn
+}
+
+// Body fills an encoder with the execute request body.
+func (req *UnprepareRequest) Body(res SchemaResolver, enc *msgpack.Encoder) error {
+	return fillUnprepare(enc, *req.stmt)
+}
+
+type ExecutePreparedRequest struct {
```

Comment:
A description for the public API is missed.

Reply:
Added.